packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -265,6 +265,26 @@ export interface ISemanticColors {
    */
   listItemBackgroundCheckedHovered: string;
 
+  /**
+   * The color of a link.
+   */
+  linkText: string;
+
+  /**
+   * The color of a hovered link. Also used when the link is active.
+   */
+  linkTextHovered: string;
+
+  /**
+   * The color of a focused link.
+   */
+  linkTextFocused: string;
```

Comment:
we don't need the focused state if it doesn't actually change color (links typically don't change color on focus anyways)

Reply:
Good point, I've removed this.